setup.py

```diff
@@ -71,7 +73,7 @@ def get_gdal_config():
         }, gdal_version_str
 
     if include_dir or library_dir or gdal_version_str:
-        log.warn(
+        logger.warn(
```

Comment:
```suggestion
        logger.warning(
```
logging.warn is deprecated...

Reply:
Whoops, didn't actually check the docs to see if the methods were the same, just checked that it printed the warning. Thanks for the catch!